spec/ruby/core/enumerable/slice_after_spec.rb

```diff
@@ -46,6 +46,15 @@
     end
   end
 
+  describe "when iterator method yields more than one value" do
+    it "should process all yielded values" do
```

Comment:
`it "processes ...`. I've always disliked the idea of using `it "should ...` as it suggests it may be OK for the code to do otherwise.